doc/user-docs/index.md

```diff
@@ -9,6 +9,8 @@ change. If you notice an error in the docs, please raise an issue. Or better yet
 
 To compile Windows Terminal yourself using the source code, follow the instructions in the [README](/README.md#developer-guidance).
 
+You check the WT [releases changes](https://github.com/microsoft/Terminal/releases).
```

Comment:
I'm not sure this line's totally necessary here. Kinda feels out of place here to me. At the very least, I'd reword it 
```suggestion
Please refer to the Windows Terminal [releases notes](https://github.com/microsoft/Terminal/releases) for the latest changes.
```